core/shared/src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -155,7 +155,7 @@ private abstract class ScalaCheckRunner extends Runner {
             case _ => new OptionalThrowable()
           }
           def fullyQualifiedName() = self.taskDef().fullyQualifiedName()
-          def selector() = new TestSelector(name)
+          def selector(): Selector = new TestSelector(name)
           def fingerprint() = self.taskDef().fingerprint()
           def duration() = -1L
```

Comment:
Out of curiosity – how come `def selector()` requires explicit type, but its neighbors (`fullyQualifiedName`, `fingerprint`, etc) do not?